src/api/routes/support/db.ts

```diff
@@ -121,6 +122,16 @@ export function getMatchingVersions (source: string, name: string, version: stri
  * Retrieve the latest available version.
  */
 export function getLatest (source: string, name: string, version?: string) {
+  return getVersionInRedis(source, name).then((data) => {
```

Comment:
This doesn't work with the version operator. If it's in Redis, searching by version will break.